doc/tutorials/resource_handlers/index.rst

```diff
@@ -0,0 +1,95 @@
+.. _resource_handlers:
+
+Adding Your Own Resource Handlers
+=================================
+
+Arcade provides a convenient way to locate asset through it's resource handlers. Arcade already has a number of
+Built-In sprites, images and other resources available for use inside the ``:resources:`` handler, which you
+may already be familiar with:
+
+.. code-block:: python
+
+   my_sprite = arcade.Sprite(":resources:images/animated_characters/female_person/femalePerson_idle.png", SPRITE_SCALE)
+
+
+Arcade also allows the ability to register additional resources handlers. This is helpful when you want to include
+your own resource folders for your project.
+
+Basic Usage
+-----------
+
+You can register a new resource handler by using ``arcade.resources.add_resource_handler(handle: str, path: Union[str, Path])``:
+
+.. code-block:: python
+
+    arcade.resources.add_resource_handler("my_resources", "/home/users/username/my_game/my_res_folder")
+
+.. note::
+
+    The ``add_resource_handler`` function must be given an **absolute** path.
+
+Then, you can use resources from your handler:
+
+.. code-block:: python
+
+    self.texture = arcade.load_texture(":my_resources:images/characters/my_character.png")
+
+Despite needing an absolute path, you can use Python's ``os.path.resolve()`` to resolve a relative path:
+
+.. code-block:: python
+
+    from pathlib import Path
+    ...
+    arcade.resources.add_resource_handler("my_resources", Path("assets/my_res_folder"))
+
+Adding Multiple Directories to a Resource Handler
+-------------------------------------------------
+
+You can also add multiple directories to a single resource handler:
+
+.. code-block:: python
+
+    # Adding multiple resources folders to the same resource handler:
+    arcade.resources.add_resource_handler("my_resources", "/home/users/username/my_game/my_first_res_folder/")
+    arcade.resources.add_resource_handler("my_resources", "/home/users/username/my_game/my_second_res_folder/")
+
+In this case, Arcade will search for a requested resources through each of the added directories, starting with the last
+added first.
```

Comment:
Perhaps specifically not that it will search `my_second_res_folder` first and `my_first_res_folder` second, makes it more clear.

Reply:
Yes, re-reading this section was not very clear. My latest commit adds some more sentences and makes things much clearer. Let me know what you think.